nixos/modules/services/web-apps/mealie.nix

```
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ...}:
+let
+  cfg = config.services.mealie;
+  pkg = pkgs.mealie;
+in
+{
+  options.services.mealie = {
+    enable = lib.mkEnableOption "Mealie, a recipe manager and meal planner";
+
+    listen_address = lib.mkOption {
```

Usually, NixOS options are in camel case.

```suggestion
    listenAddress = lib.mkOption {
```